crates/but-testing/src/command/mod.rs

```diff
@@ -469,6 +470,22 @@ pub async fn watch(args: &super::Args) -> anyhow::Result<()> {
     }
     Ok(())
 }
+pub fn watch_db(args: &super::Args) -> anyhow::Result<()> {
+    let (_repo, project) = repo_and_maybe_project(args, RepositoryOpenMode::General)?;
+    let project = project.context("Couldn't find GitButler project in directory, needed here")?;
+    let mut ctx = CommandContext::open(&project, AppSettings::default())?;
+    let db = ctx.db()?;
+    let rx = db.poll_changes(
+        ItemKind::Actions | ItemKind::Assignments | ItemKind::Workflows,
```

Comment:
I wonder - do we gain anything performance wise by not checking certain tables? For example assignments are set from the same process (the gui app) so in theory we only care about actions and workflows. Or is it just about the same amount of work?

Reply:
This command wants to check everything for testing purposes. But other subscribers can register for a subset and that will impose less work.
To me this implementation is just something that seems to work for now, and that should probably be revised once it acts up.